flare/output.py

```diff
@@ -50,59 +51,55 @@ def conclude_run(self):
         destruction function that closes all files
         """
 
-        print('-' * 20, file=self.outfiles['log'])
-        print('Run complete.', file=self.outfiles['log'])
-        for (k, v) in self.outfiles.items():
-            v.close()
-        del self.outfiles
-        self.outfiles = {}
+        logger = logging.getLogger(self.basename+'log')
+        logger.info('-' * 20)
+        logger.info('Run complete.')
+        logging.shutdown()
+        self.logger = []
 
-    def open_new_log(self, filetype: str, suffix: str):
+    def open_new_log(self, filetype: str, suffix: str, verbose='info'):
```

Comment:
Can we set the default verbosity to a lower level, e.g. 'warning' or 'error'? Otherwise almost everything in an otf run gets printed to the console.

Reply:
the screen output should be disable by the latest commit